packages/datadog_session_replay/android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,3 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="com.datadoghq.flutter">
```

Comment:
shouldn't it be `com.datadoghq.flutter.sessionreplay`?

Reply:
Apparently it should be removed entirely 🤷